jgrapht-core/src/test/java/org/jgrapht/alg/flow/PushRelabelMFImplTest.java

```diff
@@ -17,16 +17,60 @@
  */
 package org.jgrapht.alg.flow;
 
-import org.jgrapht.*;
-import org.jgrapht.alg.interfaces.*;
-import org.jgrapht.graph.*;
+import org.jgrapht.Graph;
+import org.jgrapht.alg.interfaces.MaximumFlowAlgorithm;
+import org.jgrapht.graph.DefaultEdge;
+import org.jgrapht.graph.DefaultWeightedEdge;
+import org.jgrapht.graph.DirectedWeightedMultigraph;
+import org.jgrapht.graph.SimpleDirectedGraph;
 import org.junit.Test;
 
 import static org.junit.Assert.assertEquals;
 
 public class PushRelabelMFImplTest
     extends MaximumFlowAlgorithmTest
 {
+    public static void testPushRelabelMFImpl(int graphSize){
```

Comment:
This test doesn't seem to test anything. Since this PR is a performance improvement and not a bug fix, I would recommend deleting this additional 'test'.

Reply:
What if the next update to Push-Relabel makes it slow again on this type of graphs? The issue will be closed and there will be no way of detecting there is a problem. I suggest we keep the test (as a comment is also fine).

Reply:
>> What if the next update to Push-Relabel makes it slow again on this type of graphs?
Unfortunately there's no way of preventing that. Even keeping the test as a comment won't fix that. In 6 months from now, nobody is going to remember what that test is supposed to do, or how it is supposed to perform. Unit tests can only test for errors and bugs; they unfortunately cannot test performance degradation.

Reply:
Test removed.